### PR TITLE
feat: find rows

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/grid.tsx [hunk 1]
```diff
@@ -541,6 +550,25 @@ export default class Grid {
...
+   * @param {Object|Function} conditions - object (key: column name, value: column value) or
+   *     function that check the value and returns true/false result to find rows
+   * @returns {Array} Row list
+   * @example <caption>Conditions type is object.</caption>
+   * grid.findRows({
+   *     artist: 'Birdy',
+   *     price: 10000
+   * });
+   * @example <caption>Conditions type is function.</caption>
+   * grid.findRows(function(row) {
```
Comment (seonim-ryu) [inline src/grid.tsx]: 별건 아닌데.. 예제 코드 ES6로 노출하는건 어떨까요?

Comment (jung-han) [reply src/grid.tsx]: 수정했습니다!

File: cypress/integration/data.spec.ts [hunk 2]
```diff
@@ -205,3 +206,26 @@ describe('columns', () => {
...
 });
+
+describe('rows', () => {
+  it('findRows() returns rows that meet the conditions.', () => {
+    cy.gridInstance()
+      .invoke('findRows', { name: 'Kim' })
+      .should('have.length', 1)
+      .then((res) => {
+        expect(res[0]).to.contain({ name: 'Kim', age: 10 });
+      });
```
Comment (js87zz) [inline cypress/integration/data.spec.ts]: length 체크하는 어셜션도 then절 안쪽에 넣으면 보기 더 좋을 것 같습니다.
```
cy.gridInstance()
      .invoke('findRows', { name: 'Kim' })
      .then((rows) => {
        expect(rows).to.have.length(1);
        expect(rows[0]).to.contain({ name: 'Kim', age: 10 });
      });
```

**Comment by js87zz** [comment]
리뷰 완료입니다.고생하셨습니다!

File: cypress/integration/data.spec.ts (hunk 2 above)
Comment (jung-han) [reply cypress/integration/data.spec.ts]: 수정하였습니다!

**Comment by seonim-ryu** [comment]
이견 없습니다!

**Pull request merged**